pkg/codegen/dotnet/gen_program_expressions.go

```diff
@@ -322,6 +323,9 @@ func (g *generator) GenFunctionCallExpression(w io.Writer, expr *model.FunctionC
 		g.Fgen(w, "JsonSerializer.Serialize(")
 		g.genDictionary(w, expr.Args[0])
 		g.Fgen(w, ")")
+	case "sha1":
+		// Assuming the existence of the following helper method located earlier in the preamble
+		g.Fgenf(w, "sha1(%v)", expr.Args[0])
```

Comment:
This will need to be changed as well:

```suggestion
		g.Fgenf(w, "ComputeSHA1(%v)", expr.Args[0])
```